Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -586,16 +611,26 @@ - (void)applicationWillTerminate
 - (NSMutableArray *)queue
 {
     if (!_queue) {
+#if TARGET_OS_TV
+        _queue = [[NSUserDefaults standardUserDefaults] objectForKey:SEGQueueKey];
```

Comment:
Hey @justindhill , this will need to be `_queue = [[NSUserDefaults standardUserDefaults] objectForKey:SEGQueueKey]; ?: [[NSMutableArray alloc] init];`. Without that, the queue is always `nil` since nothing has been stored for that key, and none of the payloads are added to the queue, as a result of which nothing is persisted or uploaded.